tests/parsers/fixtures/pw/failed_interrupted_relax/aiida.out

```diff
@@ -0,0 +1,926 @@
+
+     total cpu time spent up to now is        4.8 secs
+
+     WARNING: integrated charge=     8.00058489, expected=     8.00000000
+
+     total energy              =     -22.83457403 Ry
+     estimated scf accuracy    <       0.00000021 Ry
+
+     iteration #  6     ecut=    30.00 Ry     beta= 0.40
+     Davidson diagonalization with overlap
+
+---- Real-time Memory Report at c_bands before calling an iterative solver
+            91 MiB given to the printing process from OS
+             0 MiB allocation reported by mallinfo(arena+hblkhd)
+        218087 MiB available memory on the node where the printing process lives
+------------------
+     ethr =  2.65E-09,  avg # of iterations =  3.1
+
+     total cpu time spent up to now is        5.2 secs
+
+     WARNING: integrated charge=     8.00055963, expected=     8.00000000
+
+     total energy              =     -22.83457476 Ry
+     estimated scf accuracy    <       0.00000007 Ry
+
+     iteration #  7     ecut=    30.00 Ry     beta= 0.40
+     Davidson diagonalization with overlap
+
+---- Real-time Memory Report at c_bands before calling an iterative solver
+            91 MiB given to the printing process from OS
+             0 MiB allocation reported by mallinfo(arena+hblkhd)
+        218087 MiB available memory on the node where the printing process lives
+------------------
+     ethr =  9.29E-10,  avg # of iterations =  1.5
+
+     total cpu time spent up to now is        5.5 secs
+
+     WARNING: integrated charge=     8.00056842, expected=     8.00000000
+
+     total energy              =     -22.83457687 Ry
+     estimated scf accuracy    <          2.1E-09 Ry
+
+     iteration #  8     ecut=    30.00 Ry     beta= 0.40
+     Davidson diagonalization with overlap
+
+---- Real-time Memory Report at c_bands before calling an iterative solver
+            92 MiB given to the printing process from OS
+             0 MiB allocation reported by mallinfo(arena+hblkhd)
+        218087 MiB available memory on the node where the printing process lives
+------------------
+     ethr =  2.60E-11,  avg # of iterations =  3.1
+
+     total cpu time spent up to now is        5.9 secs
+
+     WARNING: integrated charge=     8.00056431, expected=     8.00000000
+
+     total energy              =     -22.83457740 Ry
+     estimated scf accuracy    <          1.4E-09 Ry
+
+     iteration #  9     ecut=    30.00 Ry     beta= 0.40
+     Davidson diagonalization with overlap
+
+---- Real-time Memory Report at c_bands before calling an iterative solver
+            92 MiB given to the printing process from OS
+             0 MiB allocation reported by mallinfo(arena+hblkhd)
+        218087 MiB available memory on the node where the printing process lives
+------------------
+     ethr =  1.79E-11,  avg # of iterations =  1.0
+
+     total cpu time spent up to now is        6.2 secs
+
+     End of self-consistent calculation
+
+          k = 0.0000 0.0000 0.0000 (   749 PWs)   bands (ev):
```

Comment:
Just a reminder to still remove occupations etc. We should really write a script for this `stdout` trimming that we can update as move more and more to the XML ^^